verifier/verifier.go

```diff
@@ -143,7 +151,7 @@ func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelop
 	}
 
 	// verify x509 trust store based authenticity
-	authenticityResult := verifyAuthenticity(ctx, trustPolicy, outcome)
+	authenticityResult := verifyAuthenticity(ctx, trustPolicy, x509TrustStore, outcome)
```

Comment:
It is simpler to have
```suggestion
	authenticityResult := verifyAuthenticity(ctx, trustPolicy, v.trustStore, outcome)
```